src/Sarif/Schemata/sarif-2.1.0-rtm.6.json

```diff
@@ -1740,6 +1744,7 @@
         "charLength": {
```

Comment:
>charLength [](http://example.com/codeflow?start=9&length=10)

for `charLength` and `byteLength`, in the spec they indeed have the default value 0.